### PR TITLE
Ft db

### PR DESCRIPTION
**Pull request by Northwind01**

### CODE REVIEW

File: opportunity/src/app/data/services/firebase.service.ts [hunk 1]
```diff
@@ -63,23 +63,41 @@ export class FirebaseCrudService {
...
+
+    //get the created object back
     const opportunity = await docRef.get();
-    console.log(opportunity.data());
+
+    //update the ngo data with the new opportunity, using the same id
+    await this.db.collection('ngos')
+      .doc(newObject.ngo.id)
+      .collection('opportunity')
+      .doc(docRef.id)
```
Comment (sneatkeen) [inline opportunity/src/app/data/services/firebase.service.ts:95]: This is a bad idea for me to store the same object in two different collections using the same id. Did you see any recommendations from firebase tutorials, do they do it in this way?

Comment (Northwind01) [reply opportunity/src/app/data/services/firebase.service.ts:95]: as it looks, the best way is to create opportunity in opp collection and nothing on ngo. When ngo whants to check its opps, it is looking in the opp collection by ngo.id

Comment (Northwind01) [reply opportunity/src/app/data/services/firebase.service.ts:95]: see the latest suggested changes

Comment (Northwind01) [reply opportunity/src/app/data/services/firebase.service.ts:95]: getAllOpportunitiesOfNGO

**Review by amandaeareias — APPROVED** [review]

**Pull request merged**